Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision 956dbfb524f0b92563b7c82f5086c69f2815342a)
+++ /dev/null	(revision 956dbfb524f0b92563b7c82f5086c69f2815342a)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: src/연습/PR.java
===================================================================
diff --git a/src/연습/PR.java b/src/연습/PR.java
deleted file mode 100644
--- a/src/연습/PR.java	(revision 956dbfb524f0b92563b7c82f5086c69f2815342a)
+++ /dev/null	(revision 956dbfb524f0b92563b7c82f5086c69f2815342a)
@@ -1,18 +0,0 @@
-package 연습;
-
-import java.util.Scanner;
-
-public class PR {
-    public static void main(String[] args) {
-// 윤년 구하기
-        Scanner sc = new Scanner(System.in);             //객체 생성 (인스턴스)
-        System.out.println("년도를 입력하시오 : ");         //입력 매서드
-        int year = sc.nextInt();                         //맴버 변수 설정.
-
-        if (year % 4 == 0 && (year % 100 != 0 || year % 400 == 0)) {
-            System.out.println("윤년입니다.");
-        } else {
-            System.out.println("윤년아닙니다.");
-        }
-    }
-}
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/java_k_digital_0814.iml\" filepath=\"$PROJECT_DIR$/java_k_digital_0814.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision 956dbfb524f0b92563b7c82f5086c69f2815342a)
+++ b/.idea/modules.xml	(date 1692249374047)
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/java_k_digital_0814.iml" filepath="$PROJECT_DIR$/java_k_digital_0814.iml" />
+      <module fileurl="file://$PROJECT_DIR$/kh_kdigital_0814_java.iml" filepath="$PROJECT_DIR$/kh_kdigital_0814_java.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
